src/golang/lib/workflow/orchestrator/orchestrator.go

```diff
@@ -28,16 +28,13 @@ const (
 	defaultTimeout = 15 * time.Minute
 )
 
-var ErrIncorrectOperatorsScheduled = errors.New("Incorrect number of operators scheduled.")
-
-func operatorExecutionError(operators map[uuid.UUID]operator.Operator, operatorId uuid.UUID) error {
-	op, ok := operators[operatorId]
-	if !ok {
-		return errors.Newf("Error during exeuction, invalid operator ID %s", operatorId)
-	}
-
-	return errors.Newf("Error during execution, operator ID %s, name %s", operatorId, op.Name)
-}
+var (
+	ErrIncorrectOperatorsScheduled = errors.New("Incorrect number of operators scheduled.")
```

Comment:
How does one schedule an incorrect number of operators? Is this an internal error and can we add a comment here?